changelog/unreleased/pr-16475.toml

```
@@ -0,0 +1,4 @@
+type = "f"
+message = "server.log now accurately reflects configured `dns_lookup_adapter_resolver_pool_size` value."
```

Nitpick: Suggesting a slightly more specific wording.
```suggestion
message = "DNS Resolver Pool warning log message now accurately reflects configured `dns_lookup_adapter_resolver_pool_size` value."
```